pkg/cache/cache_test.go

```diff
@@ -11,17 +11,17 @@ import (
 
 func TestCache(t *testing.T) {
 	t.Run("create lru expire cache test", func(t *testing.T) {
-		lruCache, err := cache.NewLRUExpireCache(1)
+		lruCache, err := cache.NewLRUExpireCache[any](1)
 		assert.NoError(t, err)
 		assert.NotNil(t, lruCache)
 
-		lruCache, err = cache.NewLRUExpireCache(0)
+		lruCache, err = cache.NewLRUExpireCache[any](0)
 		assert.ErrorIs(t, err, cache.ErrInvalidMaxSize)
 		assert.Nil(t, lruCache)
 	})
 
 	t.Run("add test", func(t *testing.T) {
-		lruCache, err := cache.NewLRUExpireCache(1)
+		lruCache, err := cache.NewLRUExpireCache[any](1)
```

Comment:
We can use `string` here to specify the type.
```suggestion
		lruCache, err := cache.NewLRUExpireCache[string](1)
```

Reply:
Thanks for comment!
I also try to use `string` here, but cache_test.go:40, `assert.Nil(t, response1)` makes error.
Error is `Expected nil, but got: ""`, so I use `any` to avoid this error.
Is there better way to solve this error?

Reply:
It seems to be possible to modify it to assert empty string instead of `assert.Nil`.

Reply:
Now, assert.Nil(t, response1) is replaced by assert.Empty(t, response1).
Thanks for nice advice!

Reply:
Thanks for review:D
`assert.NotNil` is replaced with `assert.NotEmpty`